src/GraphQL/Http/HttpResponseStreamWriter.cs

```diff
@@ -239,14 +239,14 @@ public override Task FlushAsync()
             return FlushInternalAsync(flushEncoder: true);
         }
 
-        protected override void Dispose(bool disposing)
```

Comment:
Dispose is part of the framework and shouldn't be changed to async. NET standard 2.1 has async disposable if this behavior is desirable.

Reply:
Yep not needed now.  Thanks.